x-pack/legacy/plugins/alerting/server/alerts_client.test.ts

```diff
@@ -415,6 +421,7 @@ describe('create()', () => {
           "interval": "10s",
         },
         "scheduledTaskId": "task-123",
+        "updatedAt": null,
```

Comment:
strange seeing all the `"updatedAt": null` lines, where `createdAt` is not null, but I thinks that's because the `updated_at` value in the saved objects mock is null.  Could happen IRL, so not a bad test!  But weird :-)